content/about-me/index-1.md

```diff
@@ -1,7 +1,7 @@
 ---
-aboutPageTitle:
-aboutPageDescription:
-aboutPageImage:
+aboutPageTitle: null
```

Comment:
Why was metadata added?

Reply:
I think this is a leftover after yous pushes to this branch, I will remove it

Reply:
Haha true, please remove ❤️ 
It was a test whether CMS worked